### PR TITLE
Feature/feed

### PR DESCRIPTION
**Pull request by LorenzoDrudi**
Add the following feature:

Posts' feed for all the users (both logged-in and non).

### CODE REVIEW

File: resources/views/dashboard.blade.php [hunk 1]
```diff
@@ -1,3 +1,10 @@
+@php
+    /** @var $feed @mixed */
+    $list_lambda = function ($p) { 
+        return [ 'post' => $p ];
```
Comment (FiloSanza) [inline resources/views/dashboard.blade.php:4]:
```suggestion
        return [ 'post' => $p, 'user' => $p->user()->first(), 'group' => $p->group()->first() ]; 
```

**Review by FiloSanza — CHANGES_REQUESTED** [review]
Why did you call the controller and the endpoint 'Dashboard'?
It should be either Home (just '/') or Feed ('/feed'), probably the first one works better since we would need an homepage anyway.

File: resources/views/dashboard.blade.php (hunk 1 above)
Comment (LorenzoDrudi) [reply resources/views/dashboard.blade.php]: Solved.
I've changed the name of the Controller and  I've done a new view for the home endpoint.

**Review by FiloSanza — APPROVED** [review]
image

**Pull request merged**